src/styles/blocks/body.css

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Consider adding a general `box-sizing: border-box;` rule to the `html` selector instead of `body`. This will ensure that all elements inherit this box-sizing model by default, which is a common best practice.